src/Components/ToggleSave/ToggleSave.js

```diff
@@ -1,9 +1,20 @@
-import React from 'react'
+import React, { useState } from 'react';
+import toggleOnImage from '../images/image_png.png';
+import toggleOffImage from '../images/color toggle (1).png';
+import '../ToggleSave/ToggleSave.css'
 
 const ToggleSave = () => {
+  const [isToggled, setIsToggled] = useState(false);
+
+  const handleToggle = () => {
+    setIsToggled(!isToggled);
+  };
+
```

Comment:
I think we'll need to add functionality once we connect to the API to make sure the toggle stays toggled on the savedMushroom no matter where the user is.

Reply:
Definitely! Gonna half to do some more in depth functionality and and refactoring for this to stay consistent with the mushroom card